du.md

```diff
@@ -0,0 +1,61 @@
+---
+---
+
+du
+-------
+
+The du (disk usage) command shows you the sizes of directory trees including all of their contents and sizes of individual files.
+This makes it usefull for taking down particular directories that are hogging disk space. 
+
+~~~ bash
+$ du
+92	.
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### 'du -h'
+~~~ bash
+$ du -h
+92K	.
+~~~
+
+##### Break it down
+
+-The -h option asks for 'human readable' output. This can be very usefull for people who are not as experienced programmers. The effect
```

Comment:
I think i just changed all the formatting issues, just want to make sure i got all of them?